helix-term/src/commands.rs

```diff
@@ -4206,7 +4204,7 @@ fn yank_impl(editor: &mut Editor, register: char) {
     let selections = values.len();
 
     match editor.registers.write(register, values) {
-        Ok(_) => editor.set_status(format!(
+        Ok(()) => editor.set_status(format!(
```

Comment:
Why the change in these? I prefer the originals

Reply:
Its to show that there isn't some value that can be used but just being ignored, that its matching unit. I can change it back though.